frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.jsx

```diff
@@ -224,23 +262,46 @@ export default class Smart extends Component {
             t`No change from last ${granularity}`
           ) : (
             <PreviousValueContainer gridSize={gridSize}>
-              <Variation color={changeColor}>
-                <Icon
-                  size={13}
-                  className="pr1"
-                  name={isNegative ? "arrow_down" : "arrow_up"}
-                />
-                {changeDisplay}
-              </Variation>
-              <PreviousValueVariation id="SmartScalar-PreviousValue">
-                {jt`${separator} was ${formatValue(
-                  previousValue,
-                  settings.column(column),
-                )} ${granularityDisplay}`}
-              </PreviousValueVariation>
+              <Tooltip
+                isEnabled={!canShowPreviousValue}
+                placement="bottom"
+                tooltip={
+                  <VariationTooltip>
+                    <Variation>
+                      <VariationIcon name={iconName} size={TOOLTIP_ICON_SIZE} />
+                      <VariationValue showTooltip={false}>
+                        {formatChange(lastChange)}
+                      </VariationValue>
+                    </Variation>
+
+                    {jt`${tooltipSeparator} was ${previousValueDisplay} ${granularityDisplay}`}
+                  </VariationTooltip>
+                }
+              >
+                <Variation color={changeColor}>
+                  <VariationIcon name={iconName} size={ICON_SIZE} />
+                  <VariationValue showTooltip={false}>
+                    {changeDisplay}
+                  </VariationValue>
+                </Variation>
+              </Tooltip>
+
+              {canShowPreviousValue && (
+                <PreviousValue
+                  gridSize={gridSize}
+                  id="SmartScalar-PreviousValue"
+                  responsive
+                >
+                  <PreviousValueSeparator gridSize={gridSize}>
+                    •
+                  </PreviousValueSeparator>
+
+                  {previousValueContent}
```

Comment:
nit: the tooltip uses
`{jt`${tooltipSeparator} was ${previousValueDisplay} ${granularityDisplay}`}`

but the non-tooltip version extracts those to a variable `previousValueContent`. I think both are similar and should use the same code style. e.g. either inlining both or extracting both to a variable.

Reply:
Answered here: https://github.com/metabase/metabase/pull/31722#discussion_r1245285127
Let's keep the discussion there, I will resolve this comment.

Reply:
64d5d1a71632869e49d0b5a15b99ccc74af25130